drivers/display/display_st7789v.c

```diff
@@ -437,15 +448,34 @@ static const struct display_driver_api st7789v_api = {
 	.set_orientation = st7789v_set_orientation,
 };
 
-static struct st7789v_data st7789v_data = {
+const static struct st7789v_config st7789v_config = {
 	.width = DT_INST_PROP(0, width),
 	.height = DT_INST_PROP(0, height),
+	.vcom = DT_INST_PROP(0, vcom),
+	.gctrl = DT_INST_PROP(0, gctrl),
+	.vdv_vrh_enable = (DT_INST_NODE_HAS_PROP(0, vrhs) && DT_INST_NODE_HAS_PROP(0, vdvs)),
+	.vrh_value = DT_INST_PROP(0, vrhs),
+	.vdv_value = DT_INST_PROP(0, vdvs),
+	.mdac = DT_INST_PROP(0, mdac),
+	.gamma = DT_INST_PROP(0, gamma),
+	.colmod = DT_INST_PROP(0, colmod),
+	.lcm = DT_INST_PROP(0, lcm),
+	.porch_param = DT_INST_PROP(0, porch_param),
+	.cmd2en_param = DT_INST_PROP(0, cmd2en_param),
+	.pwctrl1_param = DT_INST_PROP(0, pwctrl1_param),
+	.pvgam_param = DT_INST_PROP(0, pvgam_param),
+	.nvgam_param = DT_INST_PROP(0, nvgam_param),
+	.ram_param = DT_INST_PROP(0, ram_param),
+	.rgb_param = DT_INST_PROP(0, rgb_param),
+};
+
+static struct st7789v_data st7789v_data = {
 	.x_offset = DT_INST_PROP(0, x_offset),
 	.y_offset = DT_INST_PROP(0, y_offset),
 };
 
 PM_DEVICE_DT_INST_DEFINE(0, st7789v_pm_action);
 
 DEVICE_DT_INST_DEFINE(0, &st7789v_init,
-	      PM_DEVICE_DT_INST_REF(0), &st7789v_data, NULL, POST_KERNEL,
-	      CONFIG_DISPLAY_INIT_PRIORITY, &st7789v_api);
+		      PM_DEVICE_DT_INST_REF(0), &st7789v_data, &st7789v_config, POST_KERNEL,
```

Comment:
It is already wrapped, so please wrap it again, here and similar places, so that where it is not necessary does not exceed the 80 characters width.

Reply:
Done. I did already run uncrustify to make the code look better ;-)